codegen/src/main/scala/akka/grpc/gen/scaladsl/play/PlayScalaClientCodeGenerator.scala

```diff
@@ -20,10 +20,13 @@ object PlayScalaClientCodeGenerator extends ScalaCodeGenerator {
     b.build
   }
 
-  private val generateModule: Service => CodeGeneratorResponse.File = service => {
-    val b = CodeGeneratorResponse.File.newBuilder()
-    b.setContent(Module(service).body)
-    b.setName(s"${service.packageName.replace('.', '/')}/${service.name}Module.scala")
-    b.build
+  override def staticContent(allServices: Seq[Service]): Set[CodeGeneratorResponse.File] = {
+    if (allServices.nonEmpty) {
+      val b = CodeGeneratorResponse.File.newBuilder()
+      b.setContent(AkkaGrpcClientModule(allServices).body)
+      b.setName(s"${allServices.head.packageName.replace('.', '/')}/AkkaGrpcClientModule.scala")
```

Comment:
choosing package from the first client is maybe not ideal, another option would perhaps be least common sub package (trixy and maybe a bit surprising) and a default location (`akka.grpc.play.AkkaGrpcClientModule`) but that would make it impossible to re-use a pre-packaged client jar as the modules would have the same name.

Reply:
We should discourage 'client libraries', but perhaps still allow them if the proto codegen might contribute a lot to build time/complexity. Otherwise `akka.grpc.play.AkkaGrpcClientModule` would have been nice :)

Reply:
For example a library that doesn't really contain the gRPC as user API but rather has a separate API that somewhere inside is using Akka gRPC to talk to a service.